publication-commons/src/main/java/no/unit/nva/publication/model/business/Resource.java

```diff
@@ -199,7 +199,12 @@ private static Resource convertToResource(Publication publication) {
                    .withPublishedDate(publication.getPublishedDate())
                    .withStatus(publication.getStatus())
                    .withAssociatedArtifactsList(publication.getAssociatedArtifacts())
-                   //.withFiles() TODO: Implement this
+                   .withFilesEntries(publication.getAssociatedArtifacts().stream()
```

Comment:
Warning: we assume here that use of Resource.fromPublication today don't use file entries today